src/master-playlist-controller.js

```diff
@@ -728,16 +728,10 @@ export class MasterPlaylistController extends videojs.EventTarget {
    * removing already buffered content
    *
    * @private
+   * @deprecated
    */
   smoothQualityChange_(media = this.selectPlaylist()) {
-    if (media === this.masterPlaylistLoader_.media()) {
-      return;
-    }
-
-    this.masterPlaylistLoader_.media(media);
-
-    this.mainSegmentLoader_.resetLoader();
-    // don't need to reset audio as it is reset when media changes
+    this.fastQualityChange_(media);
```

Comment:
Should we log something the first time this is called?

Reply:
I figured since we warn on use of the option and it's an internal method, we're probably safe with just the option use log.